contracts/Staking.sol

```diff
@@ -852,4 +855,16 @@ contract Staking is
             addr := mload(add(bs, 20))
         }
     }
+
+    /// Transfers ownership of talent stake to another address.
+    ///
+    /// @dev This function requires the _talents addresses in array
+    function transferStakes(address[] memory _talents, address _newOwnerAddress) public {
+        for(uint i = 0; i < _talents.length; i++) {
+            StakeData storage stake = stakes[msg.sender][_talents[i]];
+            stakes[_newOwnerAddress][_talents[i]] = stake;
```

Comment:
We need to check if `stakes[_newOwnerAddress][_talents[i]]` doesn't exist already so we don't override an already existing stake.